.pre-commit-config.yaml

```diff
@@ -35,7 +35,7 @@ repos:
       - id: check-xml
       - id: check-yaml
         exclude: "not_rendered.yml|invalid-config.yaml"
-  - repo: https://gitlab.com/pycqa/flake8
+  - repo: https://github.com/pycqa/flake8
```

Comment:
This change has merged recently. Please rebase or merge `main` into your branch to pick up this change.